cli/tests/testdata/run/unstable_temporal_api/main.out

```diff
@@ -0,0 +1,12 @@
+Check [WILDCARD]
+Temporal.Now Instant [Temporal.Instant] {}
+Temporal.Instant Instant [Temporal.Instant] {}
+Temporal.ZonedDateTime ZonedDateTime [Temporal.ZonedDateTime] {}
+Temporal.PlainDate PlainDate [Temporal.PlainDate] {}
+Temporal.PlainTime PlainTime [Temporal.PlainTime] {}
+Temporal.PlainDateTime PlainDateTime [Temporal.PlainDateTime] {}
+Temporal.PlainYearMonth PlainYearMonth [Temporal.PlainYearMonth] {}
+Temporal.PlainMonthDay PlainMonthDay [Temporal.PlainMonthDay] {}
+Temporal.Duration Duration [Temporal.Duration] {}
+Temporal.TimeZone TimeZone [Temporal.TimeZone] {}
+Temporal.Calendar PlainDate [Temporal.PlainDate] {}
```

Comment:
Improve `console.log` integration before landing. I will work on that with @crowlKats